### DISCUSSION TITLE
Has anyone tried with Timber/WordPress?

### DISCUSSION THREAD

**Discussion by calebsmithdev** [wingsuit-designsystem/wingsuit#278] [General]
One of our struggle points has been a good way to preview and test against Twig based files in our WordPress theme. I came across this project earlier this week and I'm definitely interested in giving it a shot to see if it can work with some minor adjustments to how we handle our business logic that is Timber (framework) specific.

Just wanted to see if anyone else has tried it or has any insights before I jump in. Thanks!

**Comment by christianwiedemann** [comment]
Hi! Let me know if I can help. I will release a first beta of version 2 next week. Version 2 is storybook 7 and webpack 5 based and much more flexible. I am Drupal guy but I would love if Wingsuit works with Timber/WordPress. So if I can help - Let me know!

**Reply by calebsmithdev** [reply]
That sounds great! I was actually curious which Storybook version was being used, so that answers my question there too :) I'll give it a shot and let you know!

**Comment by calebsmithdev** [comment]
Just to follow-up on this - We ended up needing to go a more custom route to fit our needs, which was mostly using the Twing package and some custom implementations of that work. We just mocked any WordPress specific Twig/Timber functions and move the more complex stuff to a higher level, which we do not show in Storybook (similar to an ATOM design system structure).
